sklearn/utils/tests/test_validation.py

```diff
@@ -324,6 +324,45 @@ def test_check_array_dtype_warning():
     assert_equal(X_checked.format, 'csr')
 
 
+def test_check_array_accept_sparse_type_exception():
+    X = [[1, 2], [3, 4]]
+    X_csr = sp.csr_matrix(X)
+    invalid_type = SVR()
+
+    msg = "A sparse matrix was passed, but dense data is required. " \
```

Comment:
usually we would use parentheses around a string to get alignment more visually pleasing.